site/_posts/2018-07-01-jemalloc.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: "Ligthning fast memory allocation in Apache Arrow with jemalloc"
```

Comment:
Typo.

Maybe "Faster, more scalable memory allocation..."? Trying to think of there's a better way to describe the benefits than "Lightning fast"